packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -1,13 +1,36 @@
+import { makeTracer } from '@agoric/internal';
+import { makePublishKit } from '@agoric/notifier';
+
 /**
  * @import {Zone} from '@agoric/zone';
+ * @import {CctpTxEvidence} from '../types.js';
  */
 
+const trace = makeTracer('TxFeed', true);
+
+export const INVITATION_MAKERS_DESC = 'transaction oracle invitation';
+
 /**
  * @param {Zone} zone
  */
-export const prepareTransactionFeed = zone => {
-  return zone.exo('Fast USDC Feed', undefined, {});
+export const prepareTransactionFeedKit = zone => {
+  /** @type {PublishKit<CctpTxEvidence>} */
+  const { publisher, subscriber } = makePublishKit();
```

Comment:
Does this have to be durable or no?

Reply:
PublishKit is durable. At the moment it's not necessary because the only holders of its facets are not putting them into durable storage.

Reply:
Got it thanks. I just wasn't sure the difference between this and `prepareDurablePublishKit`

Reply:
🤦 you're right, that's what's necessary. nothing can be durable without baggage. Durability isn't strictly necessary for Milestone 1 but better to do it right the first time